swift/apple/FirezoneNetworkExtension/NetworkSettings.swift

```diff
@@ -9,108 +9,56 @@ import NetworkExtension
 import os.log
 
 class NetworkSettings {
-
-  // Unchanging values
-  let tunnelAddressIPv4: String
-  let tunnelAddressIPv6: String
-  let dnsAddresses: [String]
-
   // WireGuard has an 80-byte overhead. We could try setting tunnelOverheadBytes
   // but that's not a reliable way to calculate how big our packets should be,
   // so just use the minimum.
   let mtu: NSNumber = 1280
 
-  public var routes4: [NEIPv4Route] = []
-  public var routes6: [NEIPv6Route] = []
+  // Constants
+  private weak var packetTunnelProvider: NEPacketTunnelProvider?
+  private(set) var logger: AppLogger
 
   // Modifiable values
-  private(set) var resourceDomains: [String] = []
-  private(set) var matchDomains: [String] = [""]
-
-  // To keep track of modifications
-  public var hasUnappliedChanges: Bool
-
-  init(
-    tunnelAddressIPv4: String, tunnelAddressIPv6: String, dnsAddresses: [String]
-  ) {
-    self.tunnelAddressIPv4 = tunnelAddressIPv4
-    self.tunnelAddressIPv6 = tunnelAddressIPv6
-    self.dnsAddresses = dnsAddresses
-    self.hasUnappliedChanges = true
-  }
-
-  func setResourceDomains(_ resourceDomains: [String]) {
-    let sortedResourceDomains = resourceDomains.sorted()
-    if self.resourceDomains != sortedResourceDomains {
-      self.resourceDomains = sortedResourceDomains
-    }
-    self.hasUnappliedChanges = true
-  }
+  public var tunnelAddressIPv4: String?
+  public var tunnelAddressIPv6: String?
+  public var dnsAddresses: [String] = []
```

Comment:
Is there a reason these are stored as String even though the routes have a more concrete type?